.github/workflows/baseline-profile.yml

```diff
@@ -0,0 +1,73 @@
+name: Baseline profile generation
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'haze/src/androidMain/generated/baselineProfiles/baseline-prof.txt' # don't recursively update baseline profile
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'internal/baseline-profile/**'
+
+jobs:
+  baseline-profile:
+    runs-on: macos-latest
+    timeout-minutes: 45
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: 20
+
+      - uses: gradle/gradle-build-action@v2
+
+      # This allows us to build most of what we need without the emulator running
+      # and using resources
+      - name: Build sample app and benchmark
+        run: ./gradlew assembleNonMinifiedRelease
+
+      - name: Clear unused Gradle Managed Devices
+        run: ./gradlew cleanManagedDevices --unused-only
+
+      - name: Generate Baseline Profile
+        run: |
+          ./gradlew generateBaselineProfile \
+          -Pandroid.testoptions.manageddevices.emulator.gpu="swiftshader_indirect" \
```

Comment:
Unfortunately this means that we won't be using hardware acceleration, and therefore won't be going through the `RenderNode` blurring code paths on Android. I can't think of a way to handle this with GHA. I think we'll have to skip the GHA task and I'll just run it manually before releases.

Reply:
Would it work to drop Gradle Managed Devices and use this action instead https://github.com/marketplace/actions/android-emulator-runner ? Baseline profiles can be generated with connected devices instead of GMD

Reply:
No, both have the same issue. They both need to use `swiftshader_indirect` on GHA, as we don't have access to the the host GPU. [SwiftShader](https://swiftshader.googlesource.com/SwiftShader/) is a software-backed graphics API so no RenderNode AFAIK. The baseline profile only contains `ScrimImpl` which is the non-RenderNode impl.

Reply:
Thank you for the explanation 👍 I will delete this workflow.